package.json

```diff
@@ -75,25 +76,31 @@
     "execute": "node ./_scripts.js run",
     "abi:generate": "make clean && forge build --skip script --skip test --skip Base --skip Test && hardhat diamondABI",
     "typechain": "make clean && forge build --skip script --skip test --skip Base --skip Test && typechain --target ethers-v5 'out/**/*.json' --out-dir typechain",
-    "codegen": "plop"
+    "codegen": "plop",
+    "propose-safe-tx": "ts-node ./script/deploy/safe/propose-to-safe.ts",
+    "confirm-safe-tx": "ts-node ./script/deploy/safe/confirm-safe-tx.ts",
```

Comment:
A helper to load environment variables would be nice. e.g. loading the private key and selecting the chain in a dropdown.